ghost/core/core/server/data/migrations/versions/5.115/2025-03-24-07-19-27-add-identity-read-permission-to-administrators.js

```
@@ -0,0 +1,6 @@
+const {addPermissionToRole} = require('../../utils');
+
+module.exports = addPermissionToRole({
+    permission: 'Read identities',
+    role: 'Administrator',
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)